tutorials/image/getting-started/README.md

```diff
@@ -45,13 +45,15 @@ mkdir -p ./model_weights/ttj && \
 python -c "from huggingface_hub import snapshot_download; snapshot_download('ttj/sac-logos-ava1-l14-linearMSE', local_dir='./model_weights/ttj/sac-logos-ava1-l14-linearMSE', force_download=True)"
 ```
 
+Note: users can use `unzip` package (can be installed with `apt-get update && apt-get install unzip -y`) or any other compressing package
```

Comment:
Small nit: Can we move this instruction above the LAION NSFW detector section since that uses `unzip` to extract the model zip file.

```suggestion
Note: Users may need to install the `unzip` package (can be installed with `apt-get update && apt-get install unzip -y`) to extract the model files.
```